ansible/roles/common_installs/tasks/main.yml

```diff
@@ -32,7 +36,7 @@
     - htop
     - tmpreaper
     - mailutils
-    - npm
+    - nodejs
```

Comment:
I agree, this always seems to be a problem. is it worth specifying the version here e.g. `- nodejs=0.10.37` or whatever the exact version is?